lib/browser/api/crash-reporter.ts

```diff
@@ -18,7 +18,7 @@ class CrashReporter {
 
     if (submitURL == null) throw new Error('submitURL is a required option to crashReporter.start');
 
-    if (!compress) {
+    if (!compress && submitURL != '') {
```

Comment:
```suggestion
    if (!compress && uploadToServer) {
```

I think it's better to check `uploadToServer` than `submitURL`